src/frontend/src/lib/components/ui/TextWithLogo.svelte

```diff
@@ -6,14 +6,23 @@
 	export let name: string;
 	export let icon: string | undefined;
 	export let logo: 'start' | 'end' = 'end';
+	export let description: string | undefined = undefined;
 </script>
 
 <span
-	class="flex items-center"
+	class="flex"
+	class:items-center={!description}
 	class:gap-2={logo === 'start'}
 	class:gap-1={logo === 'end'}
 	class:flex-row-reverse={logo === 'start'}
 >
-	<span>{name}</span>
+	{#if description}
+		<span class="flex flex-col gap-0.5">
+			<span class="leading-5">{name}</span>
+			<span class="leading-none text-xs text-misty-rose text-left">{description}</span>
+		</span>
+	{:else}
+		<span>{name}</span>
```

Comment:
To avoid duplicating the usage of `{name}`, what do you think of following?

```
<script>
  let withDescription = false;
  $: withDescription = nonNullish(description);
</script>

<span class="flex flex-col gap-0.5">
   <span class:leading-5={withDescription}>{name}</span>
   
   {#if withDescription}
      <span class="leading-none text-xs text-misty-rose text-left">{description}</span>
		</span>
   {/else}
```

Reply:
Alternatively, can potentially `leading-5` be use for `name` without `description` as well? If so:

```
<span class="flex flex-col gap-0.5">
   <span class="leading-5">{name}</span>
   
   {#if nonNullish(description)}
      <span class="leading-none text-xs text-misty-rose text-left">{description}</span>
		</span>
   {/else}
```

Reply:
I used your second suggestion, and code-wise the component looks much neater now. Having `leading-5` for `name` affected by a few pixels the `TextWithLogo` component appearance, but it's barely noticeable. 

Local:
<img width="295" alt="local" src="https://github.com/user-attachments/assets/63861d88-b635-4ffc-9e14-f4eb79a9bb46">

Staging:
<img width="295" alt="staging" src="https://github.com/user-attachments/assets/dcb5899b-dea9-4353-847f-8d3114428a4c">